src/script/conversation/ClientMismatchHandler.ts

```diff
@@ -55,15 +55,18 @@ export class ClientMismatchHandler {
     clientMismatch: ClientMismatch,
     payload?: NewOTRMessage,
   ): Promise<NewOTRMessage> {
-    const {deleted, missing, redundant} = clientMismatch;
+    const {deleted: deletedRecipients, missing: missingRecipients, redundant: redundantRecipients} = clientMismatch;
     // Note: Broadcast messages have an empty conversation ID
```

Comment:
I would keep the notice attached to the `eventInfoEntity.conversationId !== ''` check.